### ISSUE TITLE
IX crashes chrome when resizing the window

### ISSUE THREAD

**Issue by tobiask84** [siemens/ix#1372]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/siemens/ix/blob/main/CONTRIBUTING.md).
- [X] I have not leaked any internal/restricted information like screenshots, videos, code snippets, links etc.

### What happened?

https://github.com/siemens/ix/assets/21839543/5491b84b-5d3e-4046-84ef-9c4f52406cba

I uploaded a video of it. This is on your page so you can check the versions etc. yourself. We have this problems in our page too. Resizing the page sometimes crashes chrome completely when we use the ApplicationHeader component. 

Im running:
Chrome
126.0.6478.127 (Official Build) (arm64)
on MacOs

but this issue has been reproduced by many team members.

### What type of frontend framework are you seeing the problem on?

Angular, React, JavaScript, Others

### Which version of iX do you use?

v2.3.0

### Code to produce this issue.

```shell
Please check your own website for your own code examples (the issue occurs on your page as you can see in the attached video)
```

**Comment by danielleroux** [comment]
@tobiask84 Do you have Chrome Dev Tools open?

**Comment by tobiask84** [comment]
sometimes. but when the page crashes the dev tools crash too.

**Comment by danielleroux** [comment]
We found the problem, it's related to slot element and dynamic rendering in combination with display none very strange. Affected is only chrome. We will do some final reviews tomorrow and a hotfix release

**Comment by danielleroux** [comment]
@tobiask84 Can you please install `@siemens/ix@0.0.0-pr-1373-20240705054533` to verify

**Comment by tobiask84** [comment]
I can still crash the page locally after I install that.

**Comment by danielleroux** [comment]
Please share your package-lock to be sure that the dependency are correct.

**Comment by tobiask84** [comment]
i cannot publish the package lock of internal siemens projects on github.

<img width="411" alt="image" src="https://github.com/siemens/ix/assets/21839543/7e3255f2-e345-437b-979d-256bd57b2f16">

its possible to contact me on my motius teams.

**Comment by danielleroux** [comment]
Also the ix-react. See example link if you dont know how to do it: https://stackblitz.com/edit/rfcx63?file=package.json%3AL27

**Comment by tobiask84** [comment]
Our 
<img width="555" alt="image" src="https://github.com/siemens/ix/assets/21839543/f2eea2ee-1daf-4605-a2e1-82f1bcd498fd">

**Comment by danielleroux** [comment]
> Also the ix-react. See example link if you dont know how to do it: https://stackblitz.com/edit/rfcx63?file=package.json%3AL27

⬆️ did you managed to set the snapshot also for ix-react? Otherwise the dependency will resolve always 2.3.0

**Comment by tobiask84** [comment]
Yes. changing the ix-react dependency and using your snapshot versions did fix the problem.

it works.

**Comment by tobiask84** [comment]
I reported this to the chrome dev team.

**Comment by danielleroux** [comment]
@tobiask84 also tested Chrome Canary there is no crash. Just for information here is a repository just using stencil without iX to reproduce the issue https://github.com/danielleroux/chrome-crash

I will merge the workaround pr and it will be part of the next release.

**Comment by tobiask84** [comment]
is this fix in 2.4.0? i checked npm and that seems to be the latest version
